docs/compression_algorithms/Pruning.md

```diff
@@ -27,6 +27,74 @@ Where ![L_j](https://latex.codecogs.com/png.latex?L_j) is j-th convolutional lay
 
 Then during pruning filters with smaller ![G(F_i)](https://latex.codecogs.com/png.latex?G%28F_i%29) importance function will be pruned first.
 
+#### Algorithm statistics
+A model compression can be measured by two main metrics: filter pruning level and FLOPs pruning level. While 
```

Comment:
If you open this doc now it looks ugly. The narrative breaks after Geometric Median section. It should be fixed. I also suggest moving all the auxiliary information, such as statistics description, to the bottom of the document.

Reply:
Done